force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -32,21 +32,23 @@ public with sharing class LeadTriggerHandler {
 	 */
 	public static void handleTitleNormalization(List<Lead> leadsToNormalize) {
 		for (Lead ld : leadsToNormalize) {
-			if (ld.title == 'vp' || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
+			if (ld.Title != null) {
+			if (ld.Title.containsIgnoreCase('vp') || ld.Title.containsIgnoreCase('v.p.') || ld.Title.containsIgnoreCase('vice president')) {
 				ld.Title = 'Vice President';
 			} else if (
-				ld.title.contains('mgr') ||
-				ld.title.contains('manage') ||
-				ld.title.contains('head of department')
+				ld.Title.containsIgnoreCase('mgr') ||
+				ld.Title.containsIgnoreCase('manage') ||
+				ld.Title.containsIgnoreCase('head of department')
```

Comment:
This is good but trying creatin a list that can be the values that you check against the title. That way you can update the list in one place if new values get added.